server/sonar-server/src/main/java/org/sonar/server/issue/IssueUpdater.java

```diff
@@ -287,7 +291,7 @@ public boolean setTechnicalDebt(DefaultIssue issue, @Nullable Duration value, Is
 
   public boolean setPastTechnicalDebt(DefaultIssue issue, @Nullable Duration previousTechnicalDebt, IssueChangeContext context) {
     Duration currentTechnicalDebt = issue.debt();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "debt"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)